### DISCUSSION TITLE
Setuptools 60.5.1-60.5.4 not in PyPI. Intentional?

### DISCUSSION THREAD

**Discussion by arcivanov** [pypa/setuptools#3039] [Q&A] [answered]
Changelogs are available, tags exist, but releases are not published. Is this intentional and if so why?

**Comment by webknjaz** [comment, marked as answer]
The release automation is triggered by pushing a Git tag. Then, the workflow runs tests and publishes the dists. Apparently, the integration tests failed in those workflow runs and the release job never got invoked. It's visible here: https://github.com/pypa/setuptools/actions/runs/1672078555.

FWIW this would be reasonable to track as a bug, if there's none yet.

cc @jaraco

**Reply by abravalheri** [reply]
I believe this issue got fixed recently in #3033, so the next releases should be automatically uploaded.
